src/mon/Monitor.cc

```diff
@@ -3939,31 +3939,33 @@ void Monitor::handle_command(MonOpRequestRef op)
       overall[p.first] += p.second;
     }
     f->close_section();
-
+    if(!mgr.empty()){
     mgrmon()->count_metadata("ceph_version", &mgr);
```

Comment:
- The code inside if blocks needs indentation. 
- Add space between if condition and round bracket also before curly brace.